src/Profiles/MuscleOxygen/DataPages/Base/ANTPLUS_MuscleOxygenBaseMainDataPageMsg.cpp

```diff
@@ -0,0 +1,52 @@
+#include <Profiles/MuscleOxygen/DataPages/Base/ANTPLUS_MuscleOxygenBaseMainDataPageMsg.h>
+#include <Profiles/MuscleOxygen/ANTPLUS_MuscleOxygenPrivateDefines.h>
+#include <ANTPLUS_PrivateDefines.h>
+
+
+MuscleOxygenBaseMainDataPageMsg::MuscleOxygenBaseMainDataPageMsg(uint8_t dataPageNumber) : BaseDataPageMsg<BroadcastDataMsg>()
+{
+    memset(_buffer, 0, MESSAGE_SIZE);
```

Comment:
Just curious what the benefit of memsetting the values are :) All values in the datapage should be defined so we shouldnt be left with an 0s.

Reply:
During development, you cannot be sure to have all the bytes in a defined state. Initializing with "0" makes it easier to see bugs in bit math.  Generally spoken: Initializing member data has very low cost and pays off during develplement and maintenance life cycle. In our special case "moxy" previousHemoglobinSaturation is unused and therefore a default initialization is helpful.

Reply:
Fail point, I think though I would just set it to 0 it out in the variable declaration.